defaults/main.yml

```diff
@@ -18,6 +18,10 @@ wimpy_application_pre_commands: []
 wimpy_application_post_commands: []
 wimpy_application_additional_units: []
 
+# CloudWatch logs
+wimpy_aws_cloudwatch_enabled: True
```

Comment:
Why don't disable it by default?
Imagine I want to use this role in a small project and I have a small budget (and yes, I want to use aws XD)...I don't want to check the whole documentation to know how to deactivate things...